src/pages/App.module.css

```diff
@@ -49,9 +49,6 @@ body {
   margin-bottom: 88px;
   width: 100%;
   text-align: center;
-  /* Show scrollbar always so the page doesn't jump around. */
-  overflow-y: scroll;
-  overflow-x: hidden;
```

Comment:
Not sure why this was needed but it seems kinda dangerous to remove

Reply:
yes this was the one i should have called out in the dragons section. so for some reason cypress is complaining that some buttons get hidden by a parent specifying overflow: hidden. here is the error writedown that mentions it: https://docs.cypress.io/guides/references/error-messages#cy-failed-because-the-element-cannot-be-interacted-with
we can keep the style if we think this is a dangerous change, and i can just do a `force-click` in cypress for now. what do you think?

Reply:
I wish we had the history of this file bc I don't have enough context. But I think using the force click might be better than potentially breaking something here